src/tasks/daily.ts

```diff
@@ -349,6 +398,12 @@ export function configureSnojo(): void {
 }
 
 export const DailyTasks: Task[] = [
+  {
+    name: "Chibi Buddy",
+    ready: () => have($item`ChibiBuddy™ (on)`) || have($item`ChibiBuddy™ (off)`),
+    completed: () => have($effect`ChibiChanged™`),
```

Comment:
I would use a `let` variable like you can see for `refreshedLatte` and a few other tasks; I don't think relying on the automatically-imposed limit feels particularly good. We want to be able to change or modify that limit without going through every task and deciding which one we want

Alternately, you could define a `limit` in this task itself

Reply:
I like adding a specific limit, so did that.